FreeRTOS/FreeRTOS
# shakti board support

Description
-----------
Adding board support to three SoCs from Shakti.
website - http://shakti.org.in/

Test Steps
-----------
The README in FreeRTOS/Demo/shakti give the steps to compile the image for shakti SoC

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS/issues/273

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
